src/dparse/formatter.d

```diff
@@ -3867,3 +3903,35 @@ protected:
         "postblit"
     ];
 }
+
+version (unittest)
+void testFormatNode(Node)(string sourceCode)
```

Comment:
use this new function to fix #135 and also some problems in harbored-mod (which uses dparse formatter apparently...)